tests/Operation/UpdateManyTest.php

```diff
@@ -27,6 +27,7 @@ public function testConstructorUpdateArgumentTypeCheck($update)
 
     /**
      * @dataProvider provideUpdateDocuments
+     * @doesNotPerformAssertions
```

Comment:
TIL [`@doesNotPerformAssertions`](https://phpunit.de/manual/current/en/appendixes.annotations.html#appendixes.annotations.doesNotPerformAssertions) is a thing.

Perhaps we can use this for the Bucket and WritableStream `testValidConstructorOptions()` tests?

ReadableStream's `testValidConstructorFileDocument()` test looks OK with the change you made in this PR (also because no other tests appear to call `ReadableStream::getFile()`).